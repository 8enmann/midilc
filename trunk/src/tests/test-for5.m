main(){
  Note d;
  Sequence c;
  Number i;
  c = new_sequence();
  c = c + A;
  d = A;
  for(i = 0; i < 20; i=i+1){
    if(i==4){
        continue;
    } else {
        c = c + (d .+ i);
    }
    if(i==12){
        break;
    }
  }
  play(c);
}
