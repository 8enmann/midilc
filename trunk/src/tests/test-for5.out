0,4,69
4,4,69
8,4,70
12,4,71
16,4,72
20,4,74
24,4,75
28,4,76
32,4,77
36,4,78
40,4,79
44,4,80
48,4,81

